cmd/model-registry/main.go

```diff
@@ -53,6 +54,11 @@ func main() {
 					customOption.Username,
 					customOption.Password)...),
 			)
+
+			err := service.RegisterConsumer(service.NewSimpleSerializer("application/vnd.oci.image.manifest.v1+json"))
```

Comment:
Will it cause problems? For example, if user push a docker image, will we create a ModelJob?

Reply:
I think push docker image should request klever-model-registry, WDYT?